src/nxt_php_sapi.c

```diff
@@ -1057,6 +1121,8 @@ nxt_php_execute(nxt_php_run_ctx_t *ctx, nxt_unit_request_t *r)
 
     SG(request_info).path_translated = NULL;
 
+    nxt_php_activate_user_config(*ctx);
```

Comment:
`nxt_php_execute()` called for each request and new function is also called for each request. This is the main concern about this patch - possible performance degradation.